benchopt/tests/test_benchmarks/dummy_benchmark/objective.py

```diff
@@ -16,6 +20,11 @@ def set_data(self, X, y):
         self.X, self.y = X, y
         self.lmbd = self.reg * self._get_lambda_max()
 
+    def skip(self, X, y):
+        if np.all(self.X == 0):
```

Comment:
whoops, i forgot one!

```suggestion
        if np.all(X == 0):
```